src/pages/contact/means-of-transport/index.tsx

```diff
@@ -0,0 +1,27 @@
+import DefaultLayout from '@atb/layouts/default';
+import { withGlobalData, WithGlobalData } from '@atb/layouts/global-data';
+import {
+  ContactPageLayout,
+  ContactPageLayoutProps,
+  MeansOfTransportContent,
+} from '@atb/page-modules/contact';
+import { NextPage } from 'next';
+
+export type TicketControlAndFeePageProps =
```

Comment:
Symptoms of copy-paste in this file. There should not be "ticket-control" references here